dominate/dom_tag.py

```diff
@@ -440,12 +443,17 @@ def clean_attribute(attribute):
       'phor': 'for',
     }.get(attribute, attribute)
 
+    # A list of attribute prefixes which mean that underscores should be converted to dashes
+    # This allows attributes like data_username or hx_post become data-username and hx-post
+    # hx_ prefix is for HTMX and up_ prefix is for Unpoly
+    SPECIAL_PREFIX_LIST = ('data_', 'aria_', 'up_', 'hx_')
```

Comment:
Suggestion: move this to `dominate.ATTR_PREFIXES` and add a function to modify it:

```python
ATTR_PREFIXES = ['data_', 'aria_']

def add_attr_prefix(*args: tuple[str]):
  ATTR_PREFIXES.extend(args)
```

Then, a user who wants to use htmx or unpoly does `dominate.add_attr_prefix('hx_', 'up_')`.  

This provides flexibility without the need to support a 3rd party library by name.

Global settings are often discouraged but, in this case, it makes sense IMO since you aren't always guaranteed to have a top level object to configure.  The Python stdlib also has top-level global configs for this in a number of places where they make sense.

Reply:
Maye `DASHED_ATTRS` is the right name for this.